tensorflow/core/user_ops/chol_grad.cc

```diff
@@ -0,0 +1,115 @@
+//Copyright (c) 2016, Alexander G. de G. Matthews and James Hensman. All rights reserved.
```

Comment:
You need to remove this copyright notice and replace it with the standard Apache license header.